dashboard/app/models/level.rb

```diff
@@ -216,7 +216,9 @@ def delete_custom_level_file
     ['Unplugged', # no solutions
      'TextMatch', 'Multi', 'External', 'Match', 'ContractMatch', 'LevelGroup', # dsl defined, covered in dsl
      'Applab', 'Gamelab', # all applab and gamelab are freeplay
-     'NetSim', 'Odometer', 'Vigenere', 'FrequencyAnalysis', 'TextCompression', 'Pixelation'] # widgets
+     'EvaluationQuestion', # plc evaluation
+     'NetSim', 'Odometer', 'Vigenere', 'FrequencyAnalysis', 'TextCompression', 'Pixelation',
+    ] # widgets
```

Comment:
Hmm - https://github.com/code-dot-org/code-dot-org/search?utf8=%E2%9C%93&q=TYPES_WITHOUT_IDEAL_LEVEL_SOURCE

Maybe this can be removed completely